### DISCUSSION TITLE
Import Games from "Followed Games"

### DISCUSSION THREAD

**Discussion by ThePharaohArt** [Mrxx99/Playnite.GameJoltLibrary#8] [Ideas]
Add the option to import games from Followed Games. Many free games cannot be claimed but they can be followed or added to playlists.

**Comment by Mrxx99** [comment]
That's a good idea, I also have thought of that, but this is the incentive to do it.
Created an enhancement issue here: https://github.com/Mrxx99/Playnite.GameJoltLibrary/issues/9

**Comment by Mrxx99** [comment]
@ArtexJay Importing games that are followed is now released with 1.2.0. (Enable it in the addon settings)
